.github/workflows/rust.yml

```diff
@@ -46,14 +46,17 @@ jobs:
           ~/go/pkg/mod
         key: licenses-${{ hashFiles('.github/workflows/rust.yml') }}
 
-    - name: Check licenses
+    - name: Install dependencies
       run: |
         go install github.com/google/addlicense@latest
```

Comment:
One thing that's bothering me is that the CI has been breaking lately due to updates to tools and build image.

I pinned the build image to `ubuntu-24.04` in this PR, but should we also pin all the tools (`addlicense`, `bazelisk`, `buildifier` `cargo-audit`, cargo-outdated`), and possibly even Rust toolchains for `stable` and `nightly` jobs (which would require more frequent PRs with version bumps)?